build/getting-started.md

```diff
@@ -83,23 +87,23 @@ If you built from source:
 ./build/avalanchego
 ```
 
-If you are using the released binaries:
+If you are using the pre-built binaries on Mac OSX:
 
 ```cpp
-./avalanche-<VERSION>/avalanchego
+./avalanchego-<VERSION>/build/avalanchego
 ```
 
-If you want to be able to make API calls to your node from other machines, include argument `--http-host=` \(e.g. `./build/avalanchego --http-host=`\)
+If you are using the pre-built binaries on Linux:
 
-You can use `Ctrl + C` to kill the node.
-
-To connect to the Fuji Testnet instead, use argument `--network-id=fuji`. You can get funds on the Testnet from the [faucet.](https://faucet.avax-test.network/)
+```cpp
+./avalanchego-<VERSION>-linux.4/avalanchego
```

Comment:
I think the `.4` is some kind of typo in the 1.0.6 release, so we'll probably remove that. Checking about that now

Reply:
I noticed that, but it's what's in the release archives right now. Also, the Mac version has files inside the build/ directory, Linux does not. It could use some standardization to simplify things.

I would leave it as it is now, not to confise people following the tutorial, and for future releases coordinate with the Ops to make each platform follow the same convention.